airflow/cli/commands/pool_command.py

```diff
@@ -63,13 +64,14 @@ def pool_delete(args):
 @cli_utils.action_logging
 def pool_import(args):
     """Imports pools from the file"""
-    api_client = get_current_api_client()
-    if os.path.exists(args.file):
-        pools = pool_import_helper(args.file)
-    else:
+    if not os.path.exists(args.file):
         print("Missing pools file.")
-        pools = api_client.get_pools()
+        sys.exit(1)
+    pools, failed = pool_import_helper(args.file)
     print(_tabulate_pools(pools=pools, tablefmt=args.output))
+    if len(failed) > 0:
+        print("Failed to update pool(s): {}".format(", ".join(failed)))
+        sys.exit(1)
```

Comment:
```suggestion
        exit("Failed to update pool(s): {}".format(", ".join(failed)))
```

This way the error goes to stderr, not stdout

Reply:
I see that we use a mixture of `sys.exit` and `raise SystemExit` across the cli module. Do we prefer one or the other?

Reply:
I picked `sys.exit` arbitrarily for now. We can pick one invocation and use it consistently throughout the module separately.